docs/model_library/operational_water_management/index.rst

```diff
@@ -359,68 +359,68 @@ The default objective function for this produced water operational model is to m
         \textcolor{red}{C^{TotalPiping}}+\textcolor{red}{C^{TotalStorage}}+\textcolor{red}{C^{TotalPadStorage}}+ \textcolor{red}{C^{TotalTrucking}}+\textcolor{red}{C^{Slack}-R^{TotalStorage}}
 
 
-**Completions Pad Demand Balance:** ∀p ∈ CP, t ∈ T
+**Completions Pad Demand Balance:** :math:`\forall p \in CP, t \in T`
```

Comment:
```suggestion
**Completions Pad Demand Balance:** :math:`\forall \textcolor{blue}{p \in CP}, \textcolor{blue}{t \in T}`
```

If you want extra credit, you can add color to all of these statements like we have done in the strategic model documentation.